docs/getting-started/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Getting started with Apache Gravitino"
+slug: /getting-started
+license: "This software is licensed under the Apache License version 2."
+---
+
+There are several options for getting started with Apache Gravitino.
+
+<!--Docker option-->
+Installing and configuring Hive and Trino can be a little complex.
+If you are unfamiliar with the technologies, using Docker might be a good choice.
+There are pre-packaged containers for Gravitino, Apache Hive, Apache Hadoop,
+Trino, MySQL, PostgesSQL, and others,
+check [installing Gravitino playground](./playground.md) for more details.
+
+<!--Build from source-->
+This page guides you through the process to download and install Gravitino
+from source.
+
+1. [Prepare environment](#environment-preparation)
+   - Deploy and run Gravitino on [Amazon Web Service (AWS)](#aws)
+   - Deploy and run Gravitino on [Google Compute Platform (GCP)](#gcp)
+   - Run Gravitino on [your own machine](#local-workstation)
+1. [Install Gravitino](#install-gravitino)
+1. [Start Gravitino](#start-gravitino)
+1. [Install Apache Hive](#install-apache-hive)
+1. [Interact with Apache Gravitino API](#interact-with-apache-gravitino-api)
+
+:::note
+If you want to access the instance remotely, be sure to read
+[Accessing Gravitino on AWS externally](./aws-remote-access.md).
+:::
+
+## Environment preparation
+
+### AWS
+
+To work in an AWS environment, follow these steps:
+
+1. In the AWS console, launch a new instance.
+   Select `Ubuntu` as the operating system and `t2.xlarge` as the instance type.
+   Create a key pair named *Gravitino.pem* for SSH access and download it.
+   Allow HTTP and HTTPS traffic if you want to connect to the instance remotely.
+   Set the Elastic Block Store storage to 20GiB.
+   Leave all other settings as their defaults.
+   Other operating systems and instance types may work, but are not be fully tested.
```

Comment:
"may work but have not been fully tested."

Reply:
Okay.